pytest.ini

```diff
@@ -2,4 +2,10 @@
 testpaths = scico/test docs
 addopts = --doctest-glob="*rst"
 doctest_optionflags = NORMALIZE_WHITESPACE NUMBER
-filterwarnings = ignore::DeprecationWarning:.*.compat
+filterwarnings =
+    ignore::DeprecationWarning:.*pkg_resources.*
+    ignore::DeprecationWarning:.*flax.*
+    ignore::DeprecationWarning:.*.tensorboardx.*
+    ignore::DeprecationWarning:.*xdesign.*
+    ignore:.*pkg_resources.*:DeprecationWarning
+    ignore:.*imp module.*:DeprecationWarning
```

Comment:
Can any of these easily be made more specific to avoid hiding future deprecation warnings from these packages?

Reply:
It's not so straightforward, but I can try.

Reply:
But I will do that some other time.